cmd/app/app.go

```diff
@@ -2,23 +2,25 @@ package app
 
 import (
 	"fmt"
-	"io"
-	"log"
-	"os"
-
 	"github.com/dundee/gdu/v4/build"
 	"github.com/dundee/gdu/v4/common"
 	"github.com/dundee/gdu/v4/device"
 	"github.com/dundee/gdu/v4/stdout"
 	"github.com/dundee/gdu/v4/tui"
 	"github.com/gdamore/tcell/v2"
 	"github.com/rivo/tview"
+	"io"
+	"log"
+	"os"
+	"runtime"
+	"strconv"
```

Comment:
The ordering of these imports is strange. Please use `go fmt`. It should be:

```Go
import (
	"fmt"
	"io"
	"log"
	"os"
	"runtime"
	"strconv"

	"github.com/dundee/gdu/v4/build"
	"github.com/dundee/gdu/v4/common"
	"github.com/dundee/gdu/v4/device"
	"github.com/dundee/gdu/v4/stdout"
	"github.com/dundee/gdu/v4/tui"
	"github.com/gdamore/tcell/v2"
	"github.com/rivo/tview"
)
```

Reply:
done